### PR TITLE
Resolve LD and synthetic listener test page errors

### PR DESCRIPTION
**Pull request by paulhowes-HMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3443

### Change description ###
Resolve issues loading common lib test page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <h1>ExUI Common Lib Test Page</h1>
-<xuilib-service-messages></xuilib-service-messages>
+<xuilib-service-message [key]="test-key" [message]="'This is a test xuilib-service-message for the ExUI Common Lib Test Page. For demo purposes it can not be hidden.'"></xuilib-service-message>
```
Comment (paul-graham) [inline src/app/app.component.html:2]: What is `[key]` binding to? `test-key` isn't a valid variable name so unlikely to be in `app.component.ts`...

Comment (paulhowes-HMCTS) [reply src/app/app.component.html:2]: It's used to show / hide messages. When a message is hidden the key is added to the session cookie in order for the message to remain hidden for the duration of the session

Comment (paul-graham) [reply src/app/app.component.html:2]: So what is `test-key`?

Comment (paulhowes-HMCTS) [reply src/app/app.component.html:2]: It's the name I gave the test-key, nothing more! It's for the test page in the lib.

Comment (udaydenduluri33) [reply src/app/app.component.html:2]: Agree with @paul-graham  tast-key is not ideal variable name. But looks like its not part this PRs change?

**Review by udaydenduluri33 — APPROVED** [review]
LGTMPh

**Review by paul-graham — APPROVED** [review]
Now looks fine with a binding to a string constant. :)

**Pull request merged**